.github/workflows/docs.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: git checkout
         uses: actions/checkout@v4
       - name: Install dependencies
-        run: pip install sphinx-design sphinxawesome-theme rstfmt
+        run: pip install -r docs/requirements.txt
       - name: Check formatting
         run: rstfmt --check -w 100 docs/source
```

Comment:
This should probably use the make file, to make sure the commands stay consistent.

Reply:
The makefile does not install the python requirements.

Reply:
The commented line is the `rstfmt` command.

Reply:
That's an option, I'll give it a try. 

And a related question as I've learned over the days: The policy docs repository uses line length of 80 for .rst files, which is also the line-length of  .md here in php-src.. What about using -w 80 for rstfmt?

Reply:
This message got lost: That's certainly possible, and while we're at it: .rst is currently with line length 100 in php-src and .md with 80. In policy the rstfmt / .rst is with line length 80 as well. What about streamlining this and use a line length of 80 as well here in php-src for .rst? The PR is a good opportunity to do it. @iluuu1994

Reply:
Updated with the make binding in CI and also took in the 80 chars line length, please see the main thread of the PR @iluuu1994